azafea/event_processors/endless/metrics/events/__init__.py

```diff
@@ -791,6 +791,48 @@ def _get_fields_from_payload(payload: GLib.Variant) -> Dict[str, Any]:
         return {'tables': payload.get_uint32()}
 
 
+class CacheHasInvalidElements(SingularEvent):
+    __tablename__ = 'cache_has_invalid_elements'
+    __event_uuid__ = 'cbfbcbdb-6af2-f1db-9e11-6cc25846e296'
+    __payload_type__ = '(tt)'
+
+    # These come in as uint64, but the values won’t reach the limit of a BIGINT (int64)
```

Comment:
We should have a justification in the comment here (and below).

Reply:
:+1: